docs/blog.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html class="no-js" lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Welcome | oVirt</title>
```

Comment:
You could update this for the blog page if you want :)

Reply:
Thank you for catching that! I'll update the title so it says blog.